Source/Core/DolphinWX/Config/GameCubeConfigPane.cpp

```diff
@@ -145,8 +146,12 @@ void GameCubeConfigPane::InitializeGUI()
 void GameCubeConfigPane::LoadGUIValues()
 {
   const SConfig& startup_params = SConfig::GetInstance();
+  // 0 is both Japanese and English
+  // Japanese is selected by a bit in a VI register also being set, English if cleared.
+  // This bit controlled by bForceNTSCJ.
+  int selected_language = startup_params.SelectedLanguage + (startup_params.bForceNTSCJ ? 0 : 1);
```

Comment:
Are you sure this is correct for languages !? english and japanese?
`startup_params.SelectedLanguage + 1 - (startup_params.bForceNTSCJ ? 1 : 0);`